.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": 'codelabz-dev'
+    "default": "codelabz-58d31"
```

Comment:
Hi, can you leave these files unchanged as it is

Reply:
sorry, I will change it and will take concern about it from now onwards